csharp/Svix/EventType.cs

```diff
@@ -43,12 +44,13 @@ public bool Archive(string eventType, string idempotencyKey = default)
             }
         }
 
-        public async Task<bool> ArchiveAsync(string eventType, string idempotencyKey = default, CancellationToken cancellationToken = default)
+        public async Task<bool> ArchiveAsync(string eventType, bool? expunge = null, string idempotencyKey = default, CancellationToken cancellationToken = default)
```

Comment:
Same with expunge, I'm thinking it should probably be part of "Options" rather than its own param?

Reply:
There's no `delete`/`archive` options yet. We could add one but it'd be changing the signature either way. Also, I feel like this is distinct enough where its own parameter is warranted.

Reply:
Yeah, but then it means breaking the API twice!

Anyhow, it should probably just be its own function anyway, so you'd have `archive`, and you'd have `expunge` as a standalone function (I now remembered that we talked about it in the original PR).

Reply:
Changed, just passing `null` everywhere for now.